daemon/response.go

```diff
@@ -192,7 +192,8 @@ func makeErrorResponder(status int) errorResponder {
 		} else {
 			res.Message = fmt.Sprintf(format, v...)
 		}
-		if status == 401 {
+		// FIXME: forbidden should use a different kind.
+		if status == 401 || status == 403 {
 			res.Kind = client.ErrorKindLoginRequired
```

Comment:
so at this point we actually would really like to know if we need to be root as well or not, it would be good to have a ErrorKindRootRequired

Reply:
That sounds like a good idea.  Would it be acceptable to do this as a follow up?

Reply:
yes, a follow up is ok, not sure this one will land before the follow up but otoh this is big enough that a follow up is a good idea anyway